docs/custom-design.rst

```diff
@@ -4,46 +4,98 @@ Custom Design
+                        </a>
+                        <br>
+
+                        <!-- Page sections -->
+
+                        <!-- First section -->
+                        <a href="https://example.readthedocs.io/en/latest#dummy-section-1">
+                            <div class="outer_div_page_results" id="hit__1">
+
+                                <!-- Section heading -->
+                                <span class="search__result__subheading">
+                                    Dummy <em>section</em> 1
+                                </span>
+
+                                <!-- Section content -->
+                                <p class="search__result__content">
+                                    ... Dummy <em>section</em> description  ...
+                                </p>
+                            </div>
+                        </a>
+                        <br class="br-for-hits">
+
+                        <!-- Second section -->
+                        <a href="https://example.readthedocs.io/en/latest#dummy_section-2">
+                            <div class="outer_div_page_results" id="hit__2">...</div>
+                        </a>
+                        <br class="br-for-hits">
+
+
+                        <!-- Sphinx domains of the page -->
+
+                        <!-- First sphinx domain -->
+                        <a href="https://example.readthedocs.io/en/latest#get--api-v3-sections-">
+                            <div class="outer_div_page_results" id="hit__3">
+                                
+                                <!-- Sphinx domain role name -->
+                                <span class="search__result__subheading">
+                                    http:get
+                                </span>
+
+                                <!-- Sphinx domain name -->
+                                <p class="search__result__content">
+                                    /api/v3/<em>sections</em>/
+                                </p>
+                            </div>
+                        </a>
+                        <br class="br-for-hits">
+
+                        <!-- Second sphinx domain -->
+                        <a href="https://example.readthedocs.io/en/latest#get--api-v3-sections-(str-username)">
+                            <div class="outer_div_page_results" id="hit__4">...</div>
+                        </a>
+                        <br class="br-for-hits">
+
+                    </div>
                 </div>
 
-                <div class="search__result__single" id="hit__4">
-                <div class="search__result__single" id="hit__5">
+                <!-- Results from other pages -->
+                <div class="search__result__single"><div>...</div></div>
+                <div class="search__result__single"><div>...</div></div>
+                <div class="search__result__single"><div>...</div></div>
             </div>
         </div>
     </div>
```

Comment:
Maybe we should delete this content until we settle on the final HTML, then we can add it back? Alternatively, we could use `literalinclude` from Sphinx to include this directly from the HTML: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Reply:
I have removed it for now and we can target these things things in a separate PR to imrpove the docs and other things.
I don't think that we can use `literalinclude` here because content is being dynamically generated and removed from the DOM.